modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -56,6 +56,7 @@ You can run the latency tests using a custom test image and image registry using
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
 -e IMAGE_REGISTRY="<custom_image_registry>" \
 -e CNF_TESTS_IMAGE="<custom_cnf-tests_image>" \
+-e FEATURES=performance \
```

Comment:
not sure if this is needed here, @SchSeba  WDTY?

Reply:
I'll wait before publishing. Am also going through it one last time against the initial recording.

Reply:
it's needed also here.
this is to run the tests not the mirroring command.